schemas/aerleon-definitions.schema.json

```diff
@@ -42,10 +62,11 @@
           "description": "Defines a named network, composed of IP addresses, IP address ranges, and references to other networks.\nAn object with the \"address\" property adds that address or address range to the network.\nAn object with the \"name\" property includes the contents of that network into this one.\nA single string also includes the content of that network into this one.",
           "type": "array",
           "items": {
-            "description": "Defines a named network, composed of IP addresses, IP address ranges, and references to other networks.\nAn object with the \"address\" property adds that address or address range to the network.\nAn object with the \"name\" property includes the contents of that network into this one.\nA single string also includes the content of that network into this one.",
             "oneOf": [
               {
                 "type": "string",
+                "title": "Network Reference",
+                "description": "Include another network by name. The referenced network must be defined in this file.",
```

Comment:
I don't believe this is accurate. You can define a network in one file and reference it in another.

Reply:
Yeah, just confirmed this works fine.